<!-- -->

<launch>
	<arg name="device" default="/dev/video0"/>
	<arg name="show"   default="false"/>
	<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" launch-prefix="xterm -e">
		<param name="video_device" value="$(arg device)" />
		<param name="image_width" value="640" />
		<param name="image_height" value="360" />
		<param name="pixel_format" value="mjpeg" />
		<param name="camera_frame_id" value="usb_cam" />
		<param name="io_method" value="mmap"/>
		<param name="framerate" value="20"/>
	</node>
	
	<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen" if="$(arg show)">
		<remap from="image" to="/usb_cam/image_raw"/>
		<param name="autosize" value="true" />
	</node>
</launch>
